### PR TITLE
Restructure

### PR DESCRIPTION
**Pull request by Falke-Design**
Restructure files:
- Change point of fire enable/disable events --> Moved to the end of the functions
- Changed a lot of file, so that they have the same structure --> enable, disable, enabled, toggle, ... create ... move ... remove. Also moved functions below they function from where it called (snapping)

Fix: #413

### CODE REVIEW

File: src/js/Edit/L.PM.Edit.CircleMarker.js
Comment (Falke-Design) [inline src/js/Edit/L.PM.Edit.CircleMarker.js:111]: @codeofsumit 
`CircleMarker` is the only Shape which checks if it was enabled to fire events. Should we remove this or add this to the other Shapes too?

Comment (codeofsumit) [reply src/js/Edit/L.PM.Edit.CircleMarker.js:63]: remove it for now and lets see if it creates problems. I guess there was a reason for this but my comment should have stated it

**Review by codeofsumit — CHANGES_REQUESTED** [review]
Will merge after merge-conflicts are resolved

**Pull request merged**
